StyleCop.Analyzers/StyleCop.Analyzers.Test/Lightup/LightupHelpersTests.cs

```diff
@@ -67,7 +67,7 @@ public void TestCreateSeparatedSyntaxListPropertyAccessor()
             Assert.ThrowsAny<InvalidOperationException>(() => LightupHelpers.CreateSeparatedSyntaxListPropertyAccessor<ParameterListSyntax, ParameterSyntax>(typeof(BaseParameterListSyntax), nameof(BaseParameterListSyntax.Parameters)));
         }
 
-        [Fact(Skip = "Validation is not currently performed for the second argument.")]
+        [Fact]
```

Comment:
:question: Do we know where the `InvalidOperationException` is getting thrown? Based on the codecov results, it's not from either of the new cases this pull request added.